ts/features/idpay/onboarding/components/ScrollDownButton.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import Animated, { ZoomIn, ZoomOut } from "react-native-reanimated";
+import IconButtonSolid from "../../../../components/ui/IconButtonSolid";
+
+type Props = {
+  isVisible?: boolean;
+  onPress: () => void;
+};
+
+const ScrollDownButton = (props: Props) => {
```

Comment:
could it be useful to add a test for this component? what do you think?

Reply:
I've added a new component, `ForceScrollDownView`, which handles all the logic. This component is more testable. See https://github.com/pagopa/io-app/pull/4541/commits/59cfbd94ba39e03d767954bbc261f6ec8f04c11c